packages/vsx-registry/src/browser/vsx-registry-contribution.ts

```diff
@@ -58,6 +60,29 @@ export class VSXRegistryContribution extends AbstractViewContribution<VSXRegistr
         });
     }
 
+    async registerColors(colors: ColorRegistry): Promise<void> {
+        colors.register(
```

Comment:
please like corresponding section of VS Code colors references

Reply:
What do you mean exactly?

Reply:
sorry, I meant "link"